internal_packages/account-sidebar/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "main": "./lib/main",
   "description": "Sidebar view that shows your account and important tags",
-  "license": "Proprietary",
+  "license": "GPLv3",
```

Comment:
the [SPDX license code](https://spdx.org/licenses/) is `GPL-3.0`... npm won't recognize `GPLv3` properly